dll/win32/browseui/shellbrowser.cpp

```diff
@@ -167,7 +167,6 @@ HRESULT WINAPI SHBindToFolder(LPCITEMIDLIST path, IShellFolder **newFolder)
 }
 
 static const TCHAR szCabinetWndClass[] = TEXT("CabinetWClass");
-static const TCHAR szExploreWndClass[] = TEXT("ExploreWClass");
```

Comment:
We should probably be using this window class somewhere. I think commenting it out might make sense in this case.

Reply:
Noted.

Reply:
Yep, please keep it comented out.